src/main/java/com/manywho/services/dummy/account/repositories/AccountRepository.java

```diff
@@ -16,8 +16,8 @@ public AccountRepository(Sql2o sql2o) {
         this.sql2o = sql2o;
     }
 
-    public Account find(String id) {
-        String sql = "SELECT * FROM account WHERE id = :id";
+    public Account find(String id ) {
+        String sql = "SELECT * FROM account WHERE id = :id where stateid = :stateid";
```

Comment:
There are two "where" in this query, also because the id is unique you don't need the satateid part.

Reply:
Removed stateid as suggested.